change/@ni-spright-components-a88786af-f8fe-498a-bc10-fe223f2ffa2f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
- This is marked as resolving the storybook issue. I think the agreement was migrating all of nimble to the separate package ultimately resolves that feautre. How should we stage / track it?
- There were concerns about developer workflow, what did you evalutate and what is future work?

Reply:
I've updated the PR description so it won't close the issue. I've updated the issue to describe the new plan, and we can use it to track the rest of the work. I will move the Nimble stories after this PR goes in.

I found that making a change in a component does now require `npm run build-components` before it gets picked up by the local, running Storybook. That's not so bad. I was more concerned that you would have to stop and restart Storybook, since that takes a relatively long time.

Reply:
In the storybook work I just did I really benefitted from having "hot reload" and got frustrated when it (fairly frequently) stopped working and I had to rerun commands. Could we keep that a possibility by starting a `npm run build-components:watch` or something like that in a separate console before running `npm run storybook`? Could you test that it works and document the workflow?

Reply:
> Could we keep that a possibility by starting a `npm run build-components:watch` or something like that in a separate console before running `npm run storybook`?

Yes, that works just fine. I've added a note about it in the Storybook package's README.